.github/workflows/pr_automations_init.yml

```diff
@@ -0,0 +1,54 @@
+# Initialises all automations related to PR events.
+#
+# See `issue_automations_init.yml` for the corresponding implementation for
+# issues.
```

Comment:
If the file doesn't exist, we can remove this comment:

```suggestion
```

Reply:
Fixed in 53142df. The file exists, the `_init.yml` suffix is incorrect.